$NetBSD: patch-ai,v 1.8 2012/04/08 00:04:12 dholland Exp $

- DEVICE_GROUP is set from the pkgsrc makefile, so avoid compiler
warnings by not defining it again when it's already set.

- configure paths for pkgsrc.

- ...?

--- policy.h.orig	2012-04-07 23:59:02.000000000 +0000
+++ policy.h
@@ -114,7 +114,9 @@
 /* group id that the device is chown()ed to. If not defined, the
  * primary group of "DEVICE_OWNER" is used.
  */
+#ifndef DEVICE_GROUP
 #define DEVICE_GROUP	"modem"
+#endif
 
 /* access mode for the line while getty has it - it should be accessible
  * by uucp / uucp, but not by others (imagine someone dialing into your
@@ -593,6 +595,21 @@
  */
 #define	MINFREESPACE 1024
 
+#undef DEFAULT_LOGIN_PROGRAM
+#define DEFAULT_LOGIN_PROGRAM "/usr/bin/login"
+
+#define CNDFILE               "dialin.config"
+
+#undef MGETTY_PID_FILE
+#define MGETTY_PID_FILE       "/var/run/mg-pid.%s"
+
+#undef FAX_LOG
+#define FAX_LOG         "/var/spool/fax/Faxlog"
+
+#undef MAILER
+#define MAILER          "/usr/sbin/sendmail"
+
+#undef DEVICE_GROUP
 /* fax machines exchange so-called "non-standard-frames" that can be 
  * used to identify what vendor and model is on the other end.
  * mgetty parses and prints this by default, because it can help 
@@ -601,3 +618,4 @@
  */
 #define FAX_NSF_PARSER
 
+#define AUTO_PPP
